slave.mk

```diff
@@ -1092,6 +1094,10 @@ stretch : $$(addprefix $(TARGET_PATH)/,$$(STRETCH_DOCKER_IMAGES)) \
 jessie : $$(addprefix $(TARGET_PATH)/,$$(JESSIE_DOCKER_IMAGES)) \
          $$(addprefix $(TARGET_PATH)/,$$(JESSIE_DBG_DOCKER_IMAGES))
 
+.PHONY: shell
+shell:
+	@$(SHELL)
```

Comment:
where does this SHELL defined? are you overriding the shell in the Makefile?

Reply:
SHELL is already defined at the beginning of the slave.mk file.

# Select bash for commands
.ONESHELL:
SHELL = /bin/bash
.SHELLFLAGS += -e

Reply:
what is the purpose to define this phony target shell?

Reply:
When the build is running, quickly get into the build container with all slave build ENV variables set from another shell.  The slave build ENVs are not set if we use 'make sonic-slave-bash' target.

Reply:
how to use this target then?

Reply:
We can invoke this target as below.

 make shell

Reply:
We can also execute the individual target as below.

make -f slave.mk target/python-wheels/sonic_py_common-1.0-py2-none-any.whl
etc.